includes/class-wc-stripe-customer.php

```diff
@@ -123,9 +123,14 @@ public function create_customer( $args = array() ) {
 				'description' => $description,
 			);
 		} else {
+			$billing_first_name = isset( $_POST['billing_first_name'] ) ? filter_var( wp_unslash( $_POST['billing_first_name'] ), FILTER_SANITIZE_STRING ) : ''; // phpcs:ignore WordPress.Security.NonceVerification
+			$billing_last_name  = isset( $_POST['billing_last_name'] ) ? filter_var( wp_unslash( $_POST['billing_last_name'] ), FILTER_SANITIZE_STRING ) : ''; // phpcs:ignore WordPress.Security.NonceVerification
+
+			$description = __( 'Name', 'woocommerce-gateway-stripe' ) . ': ' . $billing_first_name . ' ' . $billing_last_name . ' ' . __( ', Guest', 'woocommerce-gateway-stripe' );
```

Comment:
Please use a "format string" instead of concatenation, per https://codex.wordpress.org/I18n_for_WordPress_Developers#Best_Practices

Reply:
Good point, I copied the string that was already in there without much thinking.

Fixed: https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1013/commits/3ba4c4fd220ebad77fbd6826457654d6bdfa9394